DNA_RNA

```diff
@@ -0,0 +1,40 @@
+class DNA:
+    def __init__(self, sequence):
+        self.sequence = sequence
+        self.sequence_length = len(sequence)
+
+    def gc(self):
+        gc_counter = 0
+        for i in range(self.sequence_length):
+            if (self.sequence[i] == 'G') or (self.sequence[i] == 'C'):
+                 gc_counter += 1
+        gc_content = round(float((gc_counter / self.sequence_length) * 100), 1)
+        return gc_content
+
+    def reverse_complement(self):
+        dna_reverse_complement_dict = {'A': 'T', 'C': 'G', 'G': 'C', 'T': 'A'}
+        return ''.join([dna_reverse_complement_dict[base] for base in sequence[::-1]])
+
+    def transcribe(self):
+        dna_transcribe_dict = {'A': 'U', 'C': 'G', 'G': 'C', 'U': 'A'}
```

Comment:
There is a big problem with the output of this function, that is caused by this dict. Actually, what the function returns is 3'-5' sequence of the RNA, which is complementary to the DNA sequence of the Dna type object. Such representation of an RNA almost never used in molecular biology. In my opinion, transcribe function should return Rna with 5'-3' sequence identical (except T substituted to U) to the leading DNA string (which usually is the representation of the DNA sequence). Thus, I would suggest to modify the function, so that it returns an RNA with 5'-3' sequence, which indeed could be transcribed from the DNA sequence of a Dna object.

Reply:
Modified "transcribe" method

Reply:
OK, just add lower case t and u threre too.

Reply:
Already done